granadilla/forms.py

```diff
@@ -74,3 +75,31 @@ def save(self, commit=True):
     class Meta:
         model = models.LdapUser
         fields = ('phone', 'mobile_phone', 'internal_phone', 'new_photo')
+
+
+class LdapUserPassForm(forms.Form):
+    current_pass = forms.CharField(label='Current password', max_length=50, widget=forms.PasswordInput)
+    new_pass_1 = forms.CharField(label='New password', max_length=50, widget=forms.PasswordInput)
+    new_pass_2 = forms.CharField(label='New password again', max_length=50, widget=forms.PasswordInput)
+
+    def clean(self):
+        cleaned_data = super(LdapUserPassForm, self).clean()
+        new_pass_1 = cleaned_data.get("new_pass_1")
+        new_pass_2 = cleaned_data.get("new_pass_2")
+        if new_pass_2 is not None:
+            if new_pass_1 != new_pass_2:
+                raise forms.ValidationError("New password is not the same!")
```

Comment:
This string should be marked for translation, i.e ``forms.ValidationError(_("Passwords do not match"))``